<launch>
    <include file="$(find magna)/launch/complete_spawner_JA.launch">

        <arg value="false" name="multi" />
        <arg value="uav_" name="ns_prefix" />
        <arg value="sitl" name="mode" />
        <arg value="map" name="pose_frame_id" />
        <arg value="on" name="ual_server" />
        <arg value="30.0" name="ual_pub_rate" />
        <arg value="10.0" name="mavros_offboard_rate" />

        <arg value="true" name="ual_use" />
        <arg value="px4" name="autopilot" />
        <arg value="true" name="uav_manager_use" />
        <arg value="2" name="id" />
        <arg value="[37.558542, -5.931074, 7.89]" name="origin_geo" />
        <arg value="true" name="multimaster_use" />

    </include>
</launch>